src/picongpu/include/fields/laserProfiles/laserGaussianBeam.hpp

```diff
@@ -98,6 +124,12 @@ namespace picongpu
             // the radius of curvature of the beam's  wavefronts
             const float_X R_y = -FOCUS_POS * ( float_X(1.0) + ( y_R / FOCUS_POS )*( y_R / FOCUS_POS ) );
 
+            // initialize temporary variables
+            float_X etrans = float_X(0.0);
+            float_X etrans_norm = float_X(0.0);
+            PMACC_CASSERT_MSG(MODENUMBER_must_be_smaller_than_number_of_entries_in_LAGUERREMODES_vector, MODENUMBER<LAGUERREMODES_t::dim);
+            for ( uint32_t m = 0 ; m <= MODENUMBER ; ++m )
+                etrans_norm += LAGUERREMODES[m];
 
```

Comment:
I second that, but would do an `etrans_norm <= epsilon`. The epsilon could be taken from [`std::numeric_limits::epsilon`](http://en.cppreference.com/w/cpp/types/numeric_limits/epsilon)